### DISCUSSION TITLE
Clear-PnPDefaultColumnValues

### DISCUSSION THREAD

**Discussion by lesjtay** [pnp/powershell#1448] [Q&A]
Is there a help file somewhere for listpipebind in the Clear-PnPDefaultColumnValues cmdlet? Or any explanation of listpipebind for any cmdlet.

I've spent a lot of time trying to make it work. Any help would be much appreciated.

Les

**Comment by AndersRask** [comment]
I piped up about pipebind (pun intended!) before, since it was in the documentation but didn't work. I think there now is support on the ListItem cmdlets (see [this](https://github.com/pnp/powershell/issues/728) bug) but I wouldn't be surprised if it was not implemented across the rest of the cmdlets.

You are very welcome to create a bug on the issues list describing the issue, and that it would be a valuable addition on all cmdlets, but I'm afraid it could be a while before they get around to adding it on all cmdlets, so until then just specify the parameters inside your Foreach-Object instead

**Reply by lesjtay** [reply]
That is good information. Nice to know I'm not alone. Thank you Anders.
